Dockerfile

```diff
@@ -12,13 +12,14 @@ ENV DEBIAN_FRONTEND="noninteractive"
 # have the package manager scan the current repo list
 RUN apt-get update
 
-FROM base AS x64-version
-
 # add build tools including Google test and mock
 RUN ./opt/carma/scripts/install_build_tools.sh
 
+FROM base AS x64-version
+# Native build/dev environment
```

Comment:
Is it correct that this step is not doing anything anymore? Can we remove it. Or can Docker specifically target this step?

Reply:
actually just learned how it works, it builds up until the targeted stage.